website_chd_product_configurator/static/src/js/website_chd_product_configurator.js

```diff
@@ -0,0 +1,102 @@
+$(document).ready(function () {
+	$('.type_selection').each(function () {
```

Comment:
There's no need to loop[ just to attach event handlers:
```
$('.type_selection').on('event', function()
{
    //do stuff
}
```
attaches your event to all selected elements